README.md

```diff
@@ -255,25 +255,24 @@ From your view controller:
 import SwiftyDropbox
 
 func myButtonInControllerPressed() {
-
-    // Use only one of these two flows at once:
-
-    // Legacy authorization flow that grants a long-lived token.
+    // OAuth 2 code flow with PKCE that grants a short-lived token with scopes.
```

Comment:
Let's also mention that this will also make the SDK do the refresh process automatically, so people don't think it only gives short-term access.